app/controllers/api/requests_controller.rb

```diff
@@ -14,6 +14,11 @@ def create
     render json: { message: e.message }, status: 422
   end
 
+  def index
+    requests = Request.all
+    render json: { requests: requests.map { |req| Request::IndexSerializer.new(req) } }
```

Comment:
Why not just use ‘each_serializer’?

Reply:
Each serializer did not wrap the the array in requests: for some reason.
In order to allow for easier additions of further items into the response, we did this to get the wrapping.
If you have a suggestion on a better way to do it please share.